src/util/types.ts

```diff
@@ -0,0 +1,38 @@
+export type ScreeningType = {
```

Comment:
It's not a common pattern i Typescript to call all your types `Type`, and I see that you're not being consistent about that here anyway. So I would just remove the `Type` suffixes on all your types.